platform/pulpcore/app/viewsets/repository.py

```diff
@@ -114,6 +114,19 @@ class ImporterViewSet(NamedModelViewSet):
     endpoint_name = 'importers'
     filter_class = ImporterFilter
 
+    def update(self, request, *args, **kwargs):
```

Comment:
I'm hoping that we can avoid `*args, **kwargs` from the codebase entirely. We should only use it when we specifically don't know what args will be passed. If you see any else consider changing those too, although maybe as a separate PR.